src/challenges/challengeOne/3.Table/Table.test.tsx

```diff
@@ -1,8 +1,36 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Table } from "./Table";
-
-// describe("Table", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Table, TableProps } from "./Table";
+
+const testColumns: TableProps["columns"] = [
+  { property: "name", label: "Name" },
+  { property: "email", label: "Email" },
+];
+
+const testData: TableProps["data"] = [
+  { id: 1, name: "John Doe", email: "john.doe@example.com" },
+  { id: 2, name: "Jane Doe", email: "jane.doe@example.com" },
+];
+
+describe("Table", () => {
+  const renderComponent = (
+    columns: TableProps["columns"],
+    data: TableProps["data"],
+  ) => {
+    render(<Table columns={columns} data={data} />);
+  };
+
+  it("renders without error", () => {
+    renderComponent(testColumns, testData);
+    expect(screen.getByText("Name")).toBeInTheDocument();
+    expect(screen.getByText("Email")).toBeInTheDocument();
+  });
+
+  it("display all data correctly", () => {
```

Comment:
You should have separated the headers from the table data into two separate tests.